cdap-proto/src/main/java/co/cask/cdap/proto/metadata/MetadataSearchResponse.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright © 2016 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.proto.metadata;
+
+import java.util.Set;
+
+/**
+ * Denotes the response of the metadata search API.
+ */
+public class MetadataSearchResponse {
+  private final String sort;
+  private final int offset;
+  private final int size;
```

Comment:
size is more like page-size, total is the total number of matches

so ``num_pages = Math.ceil(total/size)``. Any suggestions for better names?